src/app/content/components/PrevNextBar.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import styled, { css } from 'styled-components';
+import { ChevronLeft } from 'styled-icons/boxicons-regular/ChevronLeft';
+import { ChevronRight } from 'styled-icons/boxicons-regular/ChevronRight';
+import { decoratedLinkStyle, textRegularStyle } from '../../components/Typography';
+import theme from '../../theme';
+import { AppState } from '../../types';
+import * as select from '../selectors';
+import { Book, LinkedArchiveTreeSection, Page } from '../types';
+import ContentLink from './ContentLink';
+
+// tslint:disable-next-line:variable-name
+const LeftArrow = styled(ChevronLeft)`
+  height: 3rem;
+  width: 3rem;
+`;
+
+// tslint:disable-next-line:variable-name
+const RightArrow = styled(ChevronRight)`
+  height: 3rem;
+  width: 3rem;
+`;
+
+interface PropTypes {
```

Comment:
i like to put these kind of interface next to where they're used, can we move it down above `PrevNextBar`?